File: beecrowd/1100-1199/1170-1179/1176.py
num = int(input())
for n in range(num):
    entry = int(input())
    n1, n2 = 0, 1
    for i in range(entry):
        aux = n2
        n2 += n1
        n1 = aux
    print(f"Fib({entry}) = {n1}")

File: C99/1100-1199/1170-1179/1176.c
#include <stdio.h>

int main()
{
    int num, num2;
    long long int f[100], n1 = 0, n2 = 1, aux;
    f[0] = n1;
    for (int i = 1; i < 100; i++)
    {
        f[i] = n2;
        aux = n2;
        n2 += n1;
        n1 = aux;
    }
    scanf("%d", &num);
    for (int i = 0; i < num; i++)
    {
        scanf("%d", &num2);
        printf("Fib(%d) = %lld\n", num2, f[num2]);
    }
    return 0;
}